src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx

```diff
@@ -53,7 +53,7 @@ export const ParaTimeSnapshot: FC<{ scope: SearchScope }> = ({ scope }) => {
           <Nodes scope={scope} />
         </StyledGrid>
         <StyledGrid item xs={22} md={5}>
-          {!mainToken.free && <TokenPriceCard token={mainToken} />}
+          {!noFiatValues && !mainToken.free && <TokenPriceCard token={mainToken} />}
```

Comment:
What card we gonna show for PontusX mainnet?

Reply:
There is no mainnet yet, only testnet. For testnet, they explicitly asked not to show the token price card, but show the "ask for test tokens" card. For mainnet, we will see if/when we get there.